service/sg-windows/sgservice.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/kardianos/service"
+	"os"
+)
+
+var logger service.Logger
+
+type program struct{
+	ConfigPath  string
+	SyncGateway *exec.Cmd
+}
+
+func (p *program) Start(s service.Service) error {
+	go p.run()
+	return nil
+}
+func (p *program) run() {
+	logger.Info("Starting Sync Gateway service...")
+	p.SyncGateway = exec.Command("C:\\Program Files (x86)\\Couchbase\\sync_gateway.exe", p.ConfigPath)
+	err := p.SyncGateway.Start()
+	if err != nil {
+		logger.Errorf("Failed to start Sync Gateway due to error %v", err)
+		return
+	}
+	completed := make(chan error, 1)
+	go func() {
+		completed <- p.SyncGateway.Wait()
+	}()
+	select {
+	case err := <- completed:
+		if err != nil {
+			logger.Errorf("Sync Gateway exitting with status = %v", err)
+		}
+	}
```

Comment:
Cheers. Pushed the change, along with convenience support for installing it as a Windows service, as well as basic operations like start, stop, restart, and uninstall.